codes/data/france/format_policy.do

```diff
@@ -97,60 +97,61 @@ g Date = date(date,"MDY",2020)
+merge m:1 date adm0 using `national', nogen update
+
 drop adm1_name //reload region name, corrupted accent due to import csv above
 merge m:1 adm1 using "data/interim/france/region_ID.dta", keep(1 3) keepusing(adm1_name adm1_pop) nogen update
 replace adm1_name = "Corse" if adm1 == 94
 replace adm1_pop = 327283 if adm1 == 94
 
 rename adm0 adm0_name
-order adm0_name adm1 adm1_name date cum_c* event_cancel ///
-event_cancel_size no_gathering no_gathering_size ///
-no_gathering_national no_gathering_national_size ///
-school_closure school_closure_size school_closure_regional school_closure_national ///
-social_distance social_distance_size 
-
+order adm0_name adm1 adm1_name date cum_c* 
 sort adm1 date
 
-
 *change no_gathering_national for multiple dummies with different cut-off
 g no_gathering_national_5000 = 1 if no_gathering_national_size == 5000
 g no_gathering_national_1000 = 1 if no_gathering_national_size == 1000
 g no_gathering_national_100 = 1 if no_gathering_national_size == 100
 drop no_gathering_national no_gathering_national_size
 
-
-
 foreach var in "event_cancel" "event_cancel_popw" "business_closure" "home_isolation" ///
-"event_cancel_size" "no_gathering" "no_gathering_popw" "no_gathering_size" ///
-"no_gathering_national_5000" "no_gathering_national_1000" "paid_sick_leave" ///
-"no_gathering_national_100" "school_closure" "school_closure_popw" "school_closure_size" ///
+"no_gathering" "no_gathering_popw" "no_gathering_national_5000" "no_gathering_national_1000" ///
+"paid_sick_leave" "no_gathering_national_100" "school_closure" "school_closure_popw" ///
 "school_closure_regional" "school_closure_national" "social_distance" "social_distance_popw" ///
-"social_distance_size" "social_distance_national" "social_distance_regional" {
+"social_distance_national" "social_distance_regional" {
 	replace `var' = 0 if `var' == .
-
-	sort adm1 date
+	sort adm1 date	
 	by adm1: replace `var' = sum(`var')
-}
-
-*keep dummy equals to 1 even if the policy is reinforced
-
-foreach var in "event_cancel" "event_cancel_popw" "business_closure" "home_isolation" ///
-"event_cancel_size" "no_gathering" "no_gathering_popw" "no_gathering_size" ///
-"no_gathering_national_5000" "no_gathering_national_1000" "paid_sick_leave" ///
-"no_gathering_national_100" "school_closure" "school_closure_popw" "school_closure_size" ///
-"school_closure_regional" "school_closure_national" "social_distance" "social_distance_popw" ///
-"social_distance_size" "social_distance_national" "social_distance_regional" {
 	replace `var' = 1 if `var' > 1
+	*by adm1: egen max = max(`var')
+	*by adm1: replace `var' = `var' / max
+	*drop max
 }
 
 *drop oversea regions for now
 drop if adm1 < 10
 *save
 format date %tdCCYY-NN-DD
-rename adm1_pop population
+rename (adm1_pop adm1) (population adm1_id)	
+rename *_popw *_popwt
+
+* Policy variable harmonization across country
+replace social_distance = (event_cancel + no_gathering + social_distance) /3 
```

Comment:
@sannanphan Is there a reason we are averaging these three variables here? For the other countries, we're doing this "packaging" at the time of the regression, so that these `_processed` datasets are all the most granular policies we have

Reply:
merging now but keeping this as an open conversation about harmonizing FRA policy var definitions with our standard definitions